renovate.json

```diff
@@ -54,7 +54,7 @@
       "groupName": "webdriverio packages"
     },
     {
-      "packagePatterns": ["oruga"],
+      "packagePatterns": ["@oruga-ui", "oruga"],
```

Comment:
Can we remove `oruga` as `@oruga-ui` will cover all scoped packages  ?

Reply:
@clarkdo sure! done in a new commit